Command/Command.php

```diff
@@ -10,43 +10,32 @@
 use OpenClassrooms\Bundle\OneSkyBundle\EventListener\TranslationUploadTranslationEvent;
 use OpenClassrooms\Bundle\OneSkyBundle\Model\ExportFile;
 use OpenClassrooms\Bundle\OneSkyBundle\Model\UploadFile;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command as SymfonyCommand;
 use Symfony\Component\Console\Helper\ProgressBar;
 use Symfony\Component\Console\Helper\Table;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 
 const PROGRESS_BAR_FORMAT = "<comment>%message%</comment>\n %current%/%max% [%bar%] %percent:3s%% %elapsed:6s%/%estimated:-6s% %memory:6s%";
 
 /**
  * @author Romain Kuzniak <romain.kuzniak@openclassrooms.com>
  */
-abstract class Command extends ContainerAwareCommand
+abstract class Command extends SymfonyCommand
 {
-    /**
-     * @var EventDispatcherInterface
-     */
-    protected $eventDispatcher;
-
-    /**
-     * @var ProgressBar
-     */
-    private $progressBar;
-
-    /**
-     * @return string
-     */
-    abstract protected function getCommandName();
-
-    /**
-     * @return string
-     */
-    abstract protected function getCommandDescription();
+    private ProgressBar $progressBar;
+    protected ParameterBagInterface $parameters;
+    abstract protected function getCommandName(): string;
+    abstract protected function getCommandDescription(): string;
+    public function __construct(private readonly EventDispatcherInterface $eventDispatcher, $name= null)
```

Comment:
Oh, I guess because the event dispatcher is private here now? But does it need to be?